tt_metal/hw/inc/ethernet/dataflow_api.h

```diff
@@ -154,8 +152,12 @@ void eth_send_bytes(
  */
 FORCE_INLINE
 void eth_wait_for_receiver_done() {
-    eth_send_packet(0, ((uint32_t)(&(erisc_info->bytes_sent))) >> 4, ((uint32_t)(&(erisc_info->bytes_sent))) >> 4, 1);
-    while (erisc_info->bytes_sent != 0) {
+    eth_send_packet(
+        0,
+        ((uint32_t)(&(erisc_info->user_buffer_bytes_sent))) >> 4,
+        ((uint32_t)(&(erisc_info->user_buffer_bytes_sent))) >> 4,
+        1);
+    while (erisc_info->user_buffer_bytes_sent != 0) {
```

Comment:
try to split cleanup like this into separate commits from substantive changes, otherwise leaves the reviewers eyes trying to figure out what changed

Reply:
I agree, I usually try to keep this sort of stuff more organized in my commits, but this PR got too big so I squashed a lot of things 🤡